### DISCUSSION TITLE
Quick way to import 100 .bas files

### DISCUSSION THREAD

**Discussion by Greedquest** [twinbasic/twinbasic#1244] [Q&A] [answered]
I want to migrate a project to tB, but it's taking ages to import the files one by one. There isn't even drag-and-drop or a keyboard shortcut so it's all right click menus. Any suggestions? Using the latest IDE

**Comment by WaynePhillipsEA** [comment, marked as answer]
I appreciate it's not easy at the moment.  This *should* work: Create a new project ... export it and this will create the directory structure on disk.... Copy your files into the `Sources` folder on disk, then use the import from folder option from the new project dialog in the IDE

**Reply by Greedquest** [reply]
Thanks for the speedy response I'll give it a go in a bit (no worries about the wip status of course I understand)
